bigquery/google/cloud/bigquery/model.py

```diff
@@ -25,6 +25,7 @@
 from google.api_core import datetime_helpers
 from google.cloud.bigquery import _helpers
 from google.cloud.bigquery_v2 import types
+from google.cloud.bigquery.table import EncryptionConfiguration
```

Comment:
It seems wrong for `EncryptionConfiguration` to live in the `table` module now that it's used for tables and models. Let's create a new `encryption_configuration` model for this to live. (Similar to `external_config`.)

We can make an alias in the `table` module to keep this from being a breaking change.